.github/scripts/release-workflow-package-push.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# This script is an integral part of the release workflow: .github/workflows/release.yml
+# It requires the following environment variables to function correctly:
+#
+# REQUESTED_BUILDPACK_ID - The ID of the buildpack to package and push to the container registry.
```

Comment:
If the env var is required, would it be worth checking for it explicitly and erroring out if not set?

Or alternatively, passing in as `$1` (and then assigning to a var explicitly here), so it's clearer in the GHA workflow file what is using `REQUESTED_BUILDPACK_ID`?

Reply:
I'll make the dependency more explicit. However, this also creates variables via echo that other steps need - it's a hairy ball of dependencies nonetheless. :)